editor/components/button/style.scss

```diff
@@ -7,14 +7,15 @@
 		opacity: 0.6;
 	}
 
-	&.is-toggled {
-		background: $dark-gray-500;
-		color: $white;
-	}
+	.editor-tools__tabs & {
```

Comment:
Minor: We already have some button styles in the tools stylesheet. Should we move these styles there to keep the Button component generic?